packages/svelte/src/internal/client/reactivity/batch.js

```diff
@@ -72,7 +75,10 @@ let is_flushing = false;
 
 let is_flushing_sync = false;
 
+let uid = 1;
 export class Batch {
+	id = uid++;
```

Comment:
committed by accident for debugging? doesn't seem used

Reply:
oh whoops, yeah. i keep deleting and re-adding this code